Library/Homebrew/manpages/brew.1.md

```diff
@@ -55,14 +55,12 @@ With `--verbose` or `-v`, many commands print extra debugging information. Note
   * `cat` <formula>:
     Display the source to <formula>.
 
-  * `cleanup` [`--force`] [`--prune=`<days>] [`--dry-run`] [`-s`] [<formulae>]:
+  * `cleanup` [`--prune=`<days>] [`--dry-run`] [`-s`] [<formulae>]:
```

Comment:
The following paragraph still claims “By default, does not remove out-of-date keg-only brews, as other software may link directly to specific versions.”